cluster/src/main/java/com/hartwig/pipeline/tools/Versions.java

```diff
@@ -31,6 +31,9 @@
     String BAMCOMP = "1.3";
     String PROTECT = "1.2";
 
+    // Version of VirusBreakend refers to GRIDSS version to use
+    String GRIDSS_VIRUS_BREAKEND = "2.11.1";
```

Comment:
You'll be able to consolidate this now with GRIDSS when u merge

Reply:
Might be better to leave it like this in case we want to update to the one or the other later on? Essentially these are two different tools, so specifying different version makes sense I think?